src/components/GradeBookContentContainer.tsx

```diff
@@ -117,6 +130,7 @@ function GradeBookContentContainer({ course, updateCourse }: Props) {
 						<GradeItemAccordion
 							key={index}
 							item={item}
+							course={course} // TODO: pass course through context
```

Comment:
oops forgot this was here, this gets the job done but context might be cleaner in the future (maybe look into for PAS-137?)